### ISSUE TITLE
Not working in Orchard 1.10

### ISSUE THREAD

**Issue by donaldboulton** [psenechal/PJS.Bootstrap#7]
Not working in Orchard 1.10 breaking changes in 1.10 no Orchard.jQuary changed to Orchard.Resources.

**Comment by psenechal** [comment]
Thanks for the notification. 1.10 hasn't quite been released yet and the theme hasn't been updated to work with the new version.

**Comment by pacomonfort** [comment]
I have modified the code for 1.10 , do you want me to pull request the changes?

**Comment by psenechal** [comment]
Yes...please submit a pull request and I'll take a look. Thanks.
